docs/userstyle-creation.md

```diff
@@ -8,14 +8,18 @@
 
 &nbsp;
 
+### Table of Contents
+
+<!--toc:start-->
+- [Requesting Userstyles](#requesting-userstyles)
+- [Creating Userstyles](#creating-userstyles)
+<!--toc:end-->
+
 ### Requesting Userstyles
 
-To request a website to be themed, please create a
-[Port Request](https://github.com/catppuccin/catppuccin/discussions/new?category=port-requests)
-discussion on [catppuccin/catppuccin](https://github.com/catppuccin/catppuccin).
-While this may seem odd, this ensures to keep all of our requests in one place
-and makes it easier for all people to request what they want without needing to
-learn what "Stylus" exactly is.
+To request a website to be themed, please create a [Port Request](https://github.com/catppuccin/catppuccin/discussions/new?category=port-requests)
+discussion on [catppuccin/catppuccin](https://github.com/catppuccin/catppuccin). While this may seem odd, this ensures to keep all of our requests in one place
+and makes it easier for all people to request what they want without needing to learn what "Stylus" exactly is.
```

Comment:
```suggestion
To request a website to be themed, please create a [Port Request](https://github.com/catppuccin/catppuccin/discussions/new?category=port-requests) discussion on [catppuccin/catppuccin](https://github.com/catppuccin/catppuccin). While this may seem odd, this ensures to keep all of our requests in one place and makes it easier for all people to request what they want without needing to learn what "Stylus" exactly is.
```